tests/integration/modules/test_esxi.py

```diff
@@ -264,3 +264,18 @@ def test_advanced_config(service_instance):
     )
     for h in ret:
         assert ret[h]["BufferCache.FlushInterval"] == 3000
+
+
+def test_get_dns_config(service_instance):
+    """
+    Test get dns configuration on ESXi host
+    """
+    ret = esxi.get_dns_config(
+        service_instance=service_instance,
+        datacenter_name="Datacenter",
+        cluster_name="Cluster",
+    )
+    for host in ret:
```

Comment:
**suggestion** note that this will pass without doing anything if there is nothing in `ret`, which would happen if `utils_esxi.get_hosts` returns empty. We should have either a unit test to confirm the behavior always (for instance, mocking get_hosts to return something expected/unexpected and ensure that the expected results are returned or the expected exception handling happens), or this test should probably fail when there are no results (since testing nothing is a useless test)

Reply:
done

Reply:
:thinking: I'm not sure if I'm following - I see that the other test is checking when `ret` is *supposed* to be empty, but what about the first case when we expect it to be something?

Would 

```
assert ret
for host in ret:
 ...
 ```
 
 Be appropriate/correct?